pkg/morph/client/notary.go

```diff
@@ -175,7 +176,18 @@ func (c *Client) DepositNotary(amount fixedn.Fixed8, delta uint32) (res util.Uin
 		big.NewInt(int64(amount)),
 		[]interface{}{c.acc.PrivateKey().GetScriptHash(), till})
 	if err != nil {
-		return util.Uint256{}, fmt.Errorf("can't make notary deposit: %w", err)
+		if !errors.Is(err, neorpc.ErrAlreadyExists) {
+			return util.Uint256{}, fmt.Errorf("can't make notary deposit: %w", err)
+		}
+
+		// Transaction is already in mempool waiting to be processed.
+		// This is an expected situation if we restart the service.
+		c.logger.Debug("notary deposit has already been made",
+			zap.Int64("amount", int64(amount)),
+			zap.Int64("expire_at", till),
+			zap.Uint32("vub", vub),
```

Comment:
suggest _one of_:

1. "vub" -> "valid until block" if we try to be more friendly for a user
2. "expire_at" -> "till" if we want to save the original `neo-go` context

Reply:
I have copied it from the line below, let's be consistent here.